BETA.md

```diff
@@ -32,8 +32,18 @@ For this step, `node` and `npm` are required.
 ## Step 3 - publish the server locally
 From the repo root run `sbt publishLocal`
 
+## Step 4 - add semanticdb-scalac compiler plugin to your project
+Some features of the server rely on the artifacts produced by [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac).
+Enable it in your sbt build by:
+
+- adding the setting
+
+    `addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)`
+
+- adding the `-Yrangepos` scalac flag (e.g. `scalacOptions += "-Yrangepos"`)
+
 # Usage
-In your project of choice, open `sbt` in your project and run `*:scalametaEnableCompletions`.
+In your project of choice, open `sbt` and run `*:scalametaEnableCompletions`.
 As mentioned above, this will produce the necessary metadata for the server.
 
 > **NOTE**: you will need to repeat this step every time you add a new source file to the project
```

Comment:
or run `clean` in sbt

Reply:
what do you mean? If you run `clean` you will need to re-run the command again.

Reply:
I'm personally concerned about writing docs on .compilerconfig/scalametaEnableCompletions, it's really just the bare bones to be able to develop more lsp features. I think we should consider moving to eclipse settings/bloop-config/sbt-lsp before adding instructions for non-contributors to play around with. `.compilerconfig`/scalametaEnableCompletion was never meant to be widely used.

Reply:
> If you run clean you will need to re-run the command again.

I meant exactly this, just to add it to the note:

> ou will need to repeat this step every time you add a new source file to the project _or run `clean` in sbt_

Reply:
@olafurpg I understand your concern, but I think it's ok for the audience we're targeting now (a few people to have some feedback). As I said in the original post, my intent is not to advertise this in anyway.

Incidentally, I've tested the guide on 4 colleagues of mine, with different results. I'll report back soon and tweak the instruction a little.

@laughedelic ah I understand now! It makes sense, I'll add it.